index.md

```diff
@@ -1464,8 +1464,8 @@ If the [IRCv3 `away-notify` capability](https://ircv3.net/specs/extensions/away-
 
 Servers SHOULD notify clients when a user they're interacting with is away when relevant, including sending these numerics:
 
-1. {% numeric RPL_AWAY %}, with the away message, when a PRIVMSG command is directed at the away user (not to a channel they are on).
-2. {% numeric RPL_AWAY %}, with the away message, in replies to [`WHOIS`](#whois-message) messages.
+1. {% numeric RPL_AWAY %}, with the away message, when a `PRIVMSG` command is directed at the away user (not to a channel they are on).
```

Comment:
This line should use `{% command PRIVMSG %}`, and the RPL_NOWAWAY and RPL_UNAWAY numerics a few lines above should use the numeric tag.